core/examples/luigi-sample-angular/src/app/project/miscellaneous2/miscellaneous2.component.html

```diff
@@ -1,8 +1,8 @@
 <section class="fd-section">
   <div class="fd-section__header">
-    <h1 class="fd-section__title">Miscellaneous 2</h1>
+    <h1 class="fd-section__title">A normal Node</h1>
```

Comment:
Normal node does not sound right, maybe Sample node? Or, do we need it at all?

Reply:
Removed completely. Misc2 is gone.